diff -Naur Linux/ExternalTargets.global Linux_new/ExternalTargets.global
--- Linux/ExternalTargets.global	2013-10-30 09:28:30.000000000 +0100
+++ Linux_new/ExternalTargets.global	2020-06-24 13:12:05.218051128 +0200
@@ -122,12 +122,12 @@
 	@echo XXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX
 	@echo XXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX
 ################################################### Uncommment from here
-#make -k clean_debug_x64
-#@echo $(LINE2)
-#@echo $(LINE2)
-#@echo $(LINE2)
-#@echo $(LINE2)		
-#make -k clean_release_x64
+	make -k clean_debug_x64
+	@echo $(LINE2)
+	@echo $(LINE2)
+	@echo $(LINE2)
+	@echo $(LINE2)		
+	make -k clean_release_x64
 ################################################### to here to build for X64.
 	@echo $(LINE2)	
 	@echo This is the end of the clean up process.
diff -Naur Linux/Makefile.global Linux_new/Makefile.global
--- Linux/Makefile.global	2013-10-30 09:28:32.000000000 +0100
+++ Linux_new/Makefile.global	2020-06-24 13:11:37.138052119 +0200
@@ -81,7 +81,7 @@
 DEBUG_OBJ_DIR=debug/obj
 DEBUG_LIB_DIR=debug/lib
 DEBUG_CC= g++ -c -I $(INCLUDE_DIR) -D$(TARGET_OS) -ggdb
-DEBUG_CL= g++ $(SYSTEM_LIBS) -gddb
+DEBUG_CL= g++ $(SYSTEM_LIBS) -ggdb
 
 RELEASE_EXE_DIR=release/bin
 RELEASE_OBJ_DIR=release/obj
